audio/out/ao_pulse.c

```diff
@@ -24,15 +24,15 @@
 #include <string.h>
 #include <stdint.h>
 #include <math.h>
-#include <pthread.h>
```

Comment:
This file doesn't seem to need pthreads at all in the first place.

Reply:
See https://github.com/mpv-player/mpv/pull/12709#issuecomment-1793502512 for explenation why some stray `<pthread.h>` has to be there until new win32 impl.

Reply:
But I will recheck those dummy ones.

Reply:
removed